Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -25,6 +25,11 @@ internal class ActionsTreeView : ViewBase<ActionsTreeView.ViewState>
         //save TreeView element id's of individual input actions and bindings to ensure saving of expanded state
         private Dictionary<Guid, int> m_GuidToTreeViewId;
 
+        private const string kCmd_Rename = "Rename";
```

Comment:
If these strings are really needed I rather see we use a common internal struct to keep the constants instead of duplicating them. I notice this is a duplicate of changes in ActionsMapsView?